_posts/2020-12-24-JabRef5-2.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  🎄 JabRef 5.2 Release 🎄
+id: jabref-5-2
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)"
+---
+
+We are proud to announce the release of the next milestone of JabRef, and we are releasing JabRef 5.2. Just in time for the holidays!
+We are also happy to announce that [Domninik Voigt](https://github.com/DominikVoigt), who worked on JabRef as part of his TODO thesis now joined the ranks of core developers!
+
+## Notable Changes and New Features
+
+After the release of the previous major milestone 5.1 we focused on bug fixes and some new features!
+
+This release brings some improvements and bug fixes for users who are using shared SQL databases for collaborative work and those that use LibreOffice.
+For users switching from EndNote to JabRef, we improved the importer. We fixed a bug where possible errors during the import were not shown to the user. In addition, we now added a YAML exporter to export your library in the YAML format.
+
+Our new developer implemented a feature for systematic literature search in JabRef. The feature still needs some work until it's usable for non-technical users. Therefore, it's not yet enabled in JabRef.
+
+Additionally, we also tweaked the user interface. In 5.0 we introduced the option to have a “Dark Theme” and with this release it is now possible to create your own theme by [adjusting the CSS](https://docs.jabref.org/advanced/custom-themes).
+The preferences dialog and some menu items were adjusted for consistency. For those that missed our Emacs/bash keybindings, we are happy to tell you that we reimplemented this feature. The feature can now be configured as part of the "Manage keybindings" dialog.
+
+The [JabRef Browser extension}(https://docs.jabref.org/collect/jabref-browser-extension) is now also compatible with the new version of Microsoft Edge and available for all major browsers (Firefox, Chrome, Edge, and Vivaldi)
+
+[And many more! In total, there are again many changes](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md) since our last official release!
+
+**You can get it from [FOSShub](https://www.fosshub.com/JabRef.html).**
+
+Please report any other issues that you encounter in our [bug tracker](https://github.com/JabRef/jabref/issues) (in case it was not already reported before).
```

Comment:
Suggestion:
> In case you notice any bugs or problems, please check our [bug tracker](https://github.com/JabRef/jabref/issues) and consider submitting a new bug report if nobody else has been reporting the failure before.